app/models/bpldc/basic_genre.rb

```diff
@@ -2,6 +2,14 @@
 
 module Bpldc
   class BasicGenre < Bpldc::Nomenclature
-    belongs_to :authority, inverse_of: :basic_genres, class_name: 'Bpldc::Authority'
+    belongs_to :authority, inverse_of: :basic_genres, class_name: 'Bpldc::Authority', touch: true
+
+    after_commit :clear_cache
+
+    private
+
+    def clear_cache
+      Rails.cache.delete('bpldc/nomenclatures/basic_genres')
+    end
```

Comment:
Should we make this a concern that gets mixed in to the `Nomenclature` subclasses?

Reply:
Ill see what I can do in this case